posts/month-in-mathlib-nov-2021.md

```diff
@@ -0,0 +1,92 @@
+---
+author: 'Mathlib community'
+category: 'month-in-mathlib'
+date: 2021-12-06 06:58:04+00:00
+description: ''
+has_math: true
+link: ''
+slug: month-in-mathlib-nov-2021
+tags: ''
+title: This month in mathlib (Nov 2021)
+type: text
+---
+
+This post summarizes some of the activity that happened in mathlib in November.
+
+* Geometry and algebraic topology.
+    - The series 
+      [PR #10284](https://github.com/leanprover-community/mathlib/pull/10284),
+      [PR #10297](https://github.com/leanprover-community/mathlib/pull/10297),
+      [PR #10303](https://github.com/leanprover-community/mathlib/pull/10303),
+      [PR #10328](https://github.com/leanprover-community/mathlib/pull/10328),
+      [PR #10334](https://github.com/leanprover-community/mathlib/pull/10334),
+      and [PR #10401](https://github.com/leanprover-community/mathlib/pull/10401)
+      constructs the sheafification of presheaves with respect to
+      a Grothendieck topology on a category, culminating with a construction
+      of colimits in the category `SheafedSpace`. It is an important building block in
+      setting up algebraic geometry, as well as a requirement for the
+      [Liquid Tensor Experiment](https://github.com/leanprover-community/lean-liquid) 
+      (LTE). A corollary is that the category of
+      sheaves with values in a suitable concrete abelian category is itself
+      an abelian category. This has been formalised for LTE, and should
+      appear in mathlib in the near future.
+    - Barycentric coordinates have been further studied in a series of PRs including for instance
+      [PR #10320](https://github.com/leanprover-community/mathlib/pull/10320) relating them to determinants.      
+    - [PR #10381](https://github.com/leanprover-community/mathlib/pull/10381)
+      proves the orthogonal group is generated by reflections
```

Comment:
This is odd here. What about moving `#10381` to `Group theory` and `#10320` to `Linear algebra and functional analysis`?
Maybe this section can then be renamed to `Algebraic topology` alone.

Reply:
I think this result fits well under "Geometry", so I agree with the current placement.
Also, naming this section "Algebraic Topology" alone (even after moving the orthogonal group PR) would not make sense at all, since a big chunk of this section is arguably about algebraic geometry.

Reply:
As you wish. I maintain that barycentric coordinates do not belong here.

Reply:
IMO this barycentric coordinates PR are fairly geometric. They are also fundamentally used in algebraic topology.

Reply:
But all of linear algebra is fairly geometric 🤔